tests/lti_toolbox/test_lti.py

```diff
@@ -209,6 +210,150 @@ def test_roles(self):
         )
         self.assertEqual(["student", "moderator"], lti.roles)
 
+    def test_has_any_of_roles(self):
+        # pylint: disable=W0212
```

Comment:
Maybe it's a convention across this project, but I prefer to avoid codes, as I never remember them
```suggestion
        # pylint: disable=protected-access
```